Ginger/Ginger/Functionalities/FindAndReplacePage.xaml.cs

```diff
@@ -756,9 +845,11 @@ public void ViewAction(FoundItem actionToView)
             ActionEditPage w;
             if(mContext == eContext.RunsetPage)
                 w = new ActionEditPage(act, General.RepositoryItemPageViewMode.View);
+            else if(mContext == eContext.AutomatePage)
+                w = new ActionEditPage(act, General.RepositoryItemPageViewMode.Automation);
             else if (Parent is BusinessFlow)
-                w = new ActionEditPage(act, General.RepositoryItemPageViewMode.Child, Parent as BusinessFlow);
-            else if(Parent is Activity)
+                w = new ActionEditPage(act, General.RepositoryItemPageViewMode.Standalone, Parent as BusinessFlow);
+            else if (Parent is Activity)
                 w = new ActionEditPage(act, General.RepositoryItemPageViewMode.Child, actParentActivity: Parent as Activity);// add save handeling to save activity if not null
```

Comment:
I think no matter if the Parent to save it BF or Activity it supposed to be loaded in "Child" mode no?

Reply:
the same as two above.
needed to ad another case